From 5193cddcee28a721beb589dba2e2165a71c43745 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Wed, 14 Sep 2022 16:23:13 -0400
Subject: [PATCH 2/2] build: fix bad upstream merge

The upstream merge of https://github.com/nodejs/node/commit/e27e709d3c
accidentally removed code related to systemtap and dtrace.

Signed-off-by: Stephen Gallagher <sgallagh@redhat.com>
---
 tools/install.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tools/install.py b/tools/install.py
index 4b01d67da54e1f3dd604e7523a846f6a89f69651..9d5f4a48bca2c926b3ffb3c51c070222d4f7ce7b 100755
--- a/tools/install.py
+++ b/tools/install.py
@@ -177,6 +177,11 @@ def files(action):
     else:
       output_lib = 'libnode.' + variables.get('shlib_suffix')
       action([output_prefix + output_lib], variables.get('libdir') + '/' + output_lib)
+  if 'true' == variables.get('node_use_dtrace'):
+    action(['out/Release/node.d'], 'lib/dtrace/node.d')
+
+  # behave similarly for systemtap
+  action(['src/node.stp'], 'share/systemtap/tapset/')
 
   action(['deps/v8/tools/gdbinit'], 'share/doc/node/')
   action(['deps/v8/tools/lldb_commands.py'], 'share/doc/node/')
-- 
2.37.3

